src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -136,6 +137,16 @@ function ComposerWithSuggestions({
         insertedEmojisRef.current = [];
     }, []);
 
+    const debouncedLowerIsScrollLikelyLayoutTriggered = useDebounce(
+        useCallback(() => (isScrollLikelyLayoutTriggered.current = false), []),
+        500,
```

Comment:
// Value picked experimentally

(just so nobody thinks it was chosen arbitrarily)